### PR TITLE
Reader following-manage: use withDimensions for resizing

### PR DESCRIPTION
**Pull request by samouri**
Also use `withDimension` helper to get all the resizing cruft out of following-manage.
`wrapWithMain` isn't beautiful, need to noodle on that. Might make it a HoC exported from ReaderMain

### CODE REVIEW

File: client/reader/following-manage/index.jsx [hunk 1]
```diff
@@ -180,20 +162,20 @@ class FollowingManage extends Component {
...
 		const searchPlaceholderText = translate( 'Search or enter URL to follow…' );
 		const hasFollows = followsCount > 0;
 		const showExistingSubscriptions = hasFollows && ! showMoreResults;
 		const sitesQueryWithoutProtocol = withoutHttp( sitesQuery );
 		const showFollowByUrl = this.shouldShowFollowByUrl();
 		const isFollowByUrlWithNoSearchResults = showFollowByUrl && searchResultsCount === 0;
-		const filteredRecommendedSites = reject(
-			recommendedSites,
-			site => includes( blockedSites, site.blogId ),
+		const filteredRecommendedSites = reject( recommendedSites, site =>
```
Comment (samouri) [inline client/reader/following-manage/index.jsx]: pretty changed this.

File: client/reader/following-manage/index.jsx [hunk 2]
```diff
@@ -246,35 +227,46 @@ class FollowingManage extends Component {
...
 export default connect( ( state, { sitesQuery } ) => ( {
-	searchResults: getReaderFeedsForQuery(
-		state,
-		{ query: sitesQuery, excludeFollowed: true, sort: SORT_BY_RELEVANCE },
-	),
-	searchResultsCount: getReaderFeedsCountForQuery(
-		state,
-		{ query: sitesQuery, excludeFollowed: true, sort: SORT_BY_RELEVANCE },
-	),
+	searchResults: getReaderFeedsForQuery( state, {
```
Comment (samouri) [inline client/reader/following-manage/index.jsx]: 257-->266 is just pretty

**Comment by apeatling** [comment]
@samouri Is this one still relevant?

**Comment by apeatling** [comment]
@samouri I'm going to close this, but please feel free to reopen once you have made any further updates.

**Comment by samouri** [comment]
ah, i figured it out. withDimensions should be a render prop!
I'm going to get back on this

**Pull request closed without merging**